### ISSUE TITLE
Missing geometry - converting FBX to OBJ

### ISSUE THREAD

**Issue by akalisingh** [assimp/assimp#834]
Here is a [screenshot](http://i.imgur.com/njNk3C0.png) of the original FBX (left), converted to OBJ (right), imported into Maya. Missing windows, bits of the side mirrors and the front bumper.

The [Audi FBX](https://dl.dropboxusercontent.com/u/1622301/test%20resources/audi.fbx) file.

I am setting AI_CONFIG_IMPORT_FBX_READ_ALL_MATERIALS flag to true.

The converter used is the latest revision of the master branch.

**Comment by kimkulling** [comment]
You are right. The FBX-Importer does not import these geometry-definitions. I checked the logs and some of the links used in the modes are reported to be broken. I guess this is some kind of link which is not handles correctly.

**Comment by tellypresence** [comment]
Note: model file linked above ([`audi.fbx`](https://dl.dropboxusercontent.com/u/1622301/test%20resources/audi.fbx)) no longer available

**Comment by tellypresence** [comment]
Please add label "Export" to this issue

**Comment by turol** [comment]
Bug appears to already happen in the FBX import step. Not an export bug.

**Comment by tellypresence** [comment]
Seems like this may require checking for dropped geometry in FBX importer

**Comment by tellypresence** [comment]
Closing since 3D model file no longer available for testing
